resources/api-docs/1.0/entity-permissions.md

```diff
@@ -0,0 +1,60 @@
+# Entity Permissions
+
+---
+
+- [All Entity Permissions](#all-entity-permissions)
+- [Create an Entity Permission](#create-entity-permissions)
+
+<a name="all-entity-permissions"></a>
+## All Entity Permissions
+
+You can get a list of all the entity-permissions of an entity by using the following endpoint.
+
+> {warning} Don't forget that all endpoints documented here need to be prefixed with `api/{{version}}/campaigns/{campaign.id}/`.
+
+
+| Method | URI | Headers |
+| :- |   :-   |  :-  |
+| GET/HEAD | `entities/{entity.id}/entity_permissions` | Default |
+
+### Results
+```json
+{
+    "data": [
+        {
+            "access": 1,
+            "action": 1,
+            "created_at": "2022-07-25T16:21:38.000000Z",
+            "created_by": null,
```

Comment:
Once `$this->entity` is removed, this needs to be cleaned up too (no created_by, is_private, updated_by`)